index.js

```diff
@@ -1,12 +1,11 @@
-var mutexify = require('mutexify')
-var hrtime = require('browser-process-hrtime')
```

Comment:
`process` is used in several other places so this particular module didn't seem to be adding much value.